docs/docs/process-payments-with-stripe.md

```diff
@@ -0,0 +1,99 @@
+
+### Stripe Elements
+
+Stripe Elements are prebuilt UI components that help you create checkout flows on the web. Unlike Checkout, Elements give you complete control over the payment experience. Elements require you to write both client-side and server-side code.
+
+React Stripe Elements is a wrapper around Stripe Elements that help you add Elements to React applications and manage the state and lifecycle of Elements. Documentation can be found at the [React Stripe Elements Github](https://github.com/stripe/react-stripe-elements).
+
+The Stripe documentation explains how to [accept a payment](https://stripe.com/docs/payments/accept-a-payment-charges#node) using Elements.
+
+As described in the documentation, you'll need to:
+
+1. Set up Stripe (client-side)
+2. Create a payment form with Elements (client-side)
+3. Create a token (client-side)
+4. Submit the token to your server (client-side)
+5. Create a charge with the token (server-side)
+
+Because of how Gatsby is rendered, you'll need to make sure the window is available before loading Stripe. One way to do this is using the useState hook to save an instance of stripe and the useEffect hook to load stripe if the window is not undefined. The code snippet below shows a small example of these changes. Instead of passing your API key to the `<StripeProvider />`, you'll pass the stripe variable.
+
+```javascript
+import { Elements, StripeProvider } from "react-stripe-elements"
+import CustomForm from "./CustomForm"
+
+const CheckoutForm = props => {
+  const [stripe, setStripe] = useState(null)
+  useEffect(() => {
+    if (typeof window !== undefined && typeof window.Stripe !== undefined) {
+      setStripe(window.Stripe("pk_YOUR_PUBLISHABLE_API_KEY"))
+    }
+  }, [])
+
+  return (
+    <StripeProvider stripe={stripe}>
+      <Elements>
+        <CustomForm />
+      </Elements>
+    </StripeProvider>
+  )
+}
+```
+
+## Recurring payments
+
+Stripe Billing is an excellent tool to create recurring payments, even providing the ability to charge based on usage or tiered subscription plans. The [Billing documentation](https://stripe.com/docs/billing) explains the different options to set up recurring payments. The [Subscription documentation](https://stripe.com/docs/billing/subscriptions/set-up-subscription) shows how to create subscription plans, create paymentMethods and customers, and create subscriptions.
+
+## Other resources
+
+- [Stripe website](https://stripe.com/)
+- [Stripe API documentation](https://stripe.com/docs/api)
+- [React Stripe Elements Github](https://github.com/stripe/react-stripe-elements)
+- [Gatsby E-Commerce Tutorial](https://www.gatsbyjs.org/tutorial/ecommerce-tutorial/)
```

Comment:
i would use the more specific Title (which are also used in https://www.gatsbyjs.org/tutorial/additional-tutorials/ and in the left Sidebar)

```suggestion
- [Making an e-commerce Gatsby Site with Stripe](https://www.gatsbyjs.org/tutorial/ecommerce-tutorial/)
```